src/bin/super/util.rs

```diff
@@ -472,6 +474,14 @@ async fn create_ec2_instance(
             .collect::<Vec<_>>()
             .join("");
     }
+    let mut password = hex_secret_32();
```

Comment:
this means the default is random string? Can we make the default `password` just to be sure its backward compatible